pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -101,7 +101,8 @@ python3.pkgs.buildPythonApplication {
   in
     ''
     makeWrapperArgs+=(
-      "''${gappsWrapperArgs[@]}"
+      # exclusively use values from wrapper
+      --unset QT_PLUGIN_PATH
```

Comment:
I don't understand this.

Reply:
If the user has this variable set to an incorrect value, say to qt5 libs, the app will not launch. The wrapper sets this var itself to the correct value, and so unsetting first ensures it can never be incorrect.

Reply:
AFAICS, this prevents qt plugins the user might be using (e.g. `libqt6gtk2`) from working, which is not very good.

Reply:
This also seem to break my fcitx5 IME for qutebrowser for me...

I'm not experiencing the "app will not launch" issue when I remove this, so I'm not exactly sure what it is you're seeing, but perhaps we can come up with some other solution here that doesn't hinder people from being able to type in qutebrowser?

Reply:
@nrdxp Do you think you could share your ambient `QT_PLUGIN_PATH` so I can try to debug the launch failure you get when this line is removed? Or perhaps I can try to replicate your config later if it's public on GitHub